### DISCUSSION TITLE
Webpages not updating

### DISCUSSION THREAD

**Discussion by kruger1955** [StefanoTesla/TeslaBoard#5] [General]
Hi Stefano
I have played around with the project and it looks very promising.
I would like to make a couple of changes on the webpages (for instance the rplacement of Italian to English, and alignments on some of the pages).
However, any changes  are not reflected on the webpages - it keeps just the old layout and text.  I suspect it has to do with the setup of Vite or locking?  
Any recomendations on how to resolve it.
Thanks
Bert

**Comment by StefanoTesla** [comment]
The real html Project is inside the html_pages  directory, you need to edit them, and then run the command npm run build, you can review it olso with npm run dev, but api will hide the divs. 

Once are builded, you need to copy the builded files to the data folder of the esp Project, At that moment you need to build the flash image.

Unfortunatly I don't find a way to make the web pages translatables mantaining reduced size

**Comment by StefanoTesla** [comment]
I'm working to a new release, hope you will enjoy (translations included)
![image](https://github.com/user-attachments/assets/331246ad-bf7c-4f35-a601-d717c1f2efb2)
